sdk/ml/azure-ai-ml/tests/test_configs/flows/flow_with_meta/compare.md

```diff
@@ -0,0 +1,29 @@
+| field name          | can be ignore? | flow.dag.yaml                                 | flow.meta.yaml                                 | run.yaml                     | standalone file       |
+| ------------------- | -------------- | --------------------------------------------- | ---------------------------------------------- | ---------------------------- | --------------------- |
+| additional_includes | N              | Y - part of flow definition                   | N - it's not meta                              | N - it's shared by all runs  | ? - not align with v2 |
```

Comment:
in flow.meta.yaml  column,
need to mention v2 sdk need to understand the addtional_includes in dag.yaml to build code snapshot